builtins/safe-baseline-configuration.json

```diff
@@ -0,0 +1,147 @@
+{
+  "removeElements": [
+    {
+      "namespace": "http://www.w3.org/1999/xhtml",
+      "name": "script"
+    },
+    {
+      "namespace": "http://www.w3.org/2000/svg",
+      "name": "script"
+    }
```

Comment:
We need to include frame/iframe/object/embed/svg+use

Reply:
Done.